infra/modular-examples/aws-msft-365/variables.tf

```diff
@@ -76,6 +82,12 @@ variable "psoxy_base_dir" {
   }
 }
 
+variable "bundle_version" {
```

Comment:
couldn't this just be read from a file in the root?

Reply:
yeah, probably